ServerCore/Areas/Identity/Pages/Account/Manage/template_ChangePassword.cshtml

```diff
@@ -0,0 +1,35 @@
+﻿@page
```

Comment:
Please at least add comments on these pages saying they should probably be deleted along with filing an issue to delete them

Reply:
Just deleted them